astronomy/trappist_gravity_model.proto.txt

```diff
@@ -10,7 +10,7 @@ gravity_model {
   # for life?", Vida et al., 2017.
   body {
     name                 : "Trappist-1"
-    mass                 : "0.0890 M☉"
+    mass                 : "0.0890 GM☉"
```

Comment:
Should this be `GM☉`, `(GM)☉`, `(GM)ᴺ☉`, `(𝒢ℳ)ᴺ☉`?

Reply:
It's fine the way you wrote it.

Reply:
But it should be `gravitational_parameter`, not `mass`.  How does it even parse?